### ISSUE TITLE
no account updates on submitted orders?

### ISSUE THREAD

**Issue by d-e-s-o** [alpacahq/Alpaca-API#75]
(paper trading; v1)

Is it intentional that no account updates are sent when an order is submitted/filled?

According to [the documentation](https://docs.alpaca.markets/api-documentation/web-api/streaming/#account-updates):
> This stream provides clients with updates pertaining to their brokerage accounts at Alpaca, including balance information.

Yet, I don't get an updated when I submit an order and it is filled subsequently. Shouldn't the account balance change when doing that?

When I explicitly retrieve `/v1/account` information I see a difference in cash:
55279.74 USD beforehand, 54987.27 USD now

I have the following code for streaming updates:
```py
import alpaca_trade_api as tradeapi

conn = tradeapi.StreamConn()

@conn.on(r'account_updates')
async def on_account_updates(conn, channel, account):
    print('account', account)


# blocks forever
conn.run(['account_updates'])
```

**Comment by ttt733** [comment]
Hello, currently, we're not sending account updates with order updates. This is something that we would like to address, but it's not a feature we've had a chance to implement yet.

**Comment by still-dreaming-1** [comment]
This would be the most common reason an account would "update". Otherwise if we are tracking how much tradable cash or buying power is in the account, we either need to calculate that manually after every every trade comes through, or call out to the API to do a GET to download the account, which kind of defeats the point of subscribing to account updates.

**Comment by dans-acc** [comment]
> This would be the most common reason an account would "update". Otherwise if we are tracking how much tradable cash or buying power is in the account, we either need to calculate that manually after every trade comes through, or call out to the API to do a GET to download the account, which kind of defeats the point of subscribing to account updates. Don't most algorithms need to know how much is in the account to help them decide how many shares to buy of a particular stock?
> 
> I guess this is just a theoretical concern for me at the moment, as I am now realizing the strategy I am currently exploring works better without streaming anyways.

I completely agree with this. I think that publishing events relating to our account would drastically improve and ease the development of algos.

**Comment by d-e-s-o** [comment]
Meanwhile, in typical underhand fashion in a commit talking about [something completely different](https://github.com/alpacahq/user-docs/commit/4898d607be071eedda21afc2121d2dbf81363d58), the section about account updates was removed from the docs. So expect that within half a year it will be removed entirely. No pull request, no review, no rationale provided, no discussion, no announcement, no deprecation notice, nothing. Alpaca. And not even that they manage to do properly for a change and left half the references to account updates in place:
![account_updates](https://user-images.githubusercontent.com/6610056/102157196-e9b2de80-3e33-11eb-9c19-11549fb229cb.png)

Un-be-lie-va-ble.

**Comment by jasonbohne123** [comment]
> Meanwhile, in typical underhand fashion in a commit talking about [something completely different](https://github.com/alpacahq/user-docs/commit/4898d607be071eedda21afc2121d2dbf81363d58), the section about account updates was removed from the docs. So expect that within half a year it will be removed entirely. No pull request, no review, no rationale provided, no discussion, no announcement, no deprecation notice, nothing. Alpaca. And not even that they manage to do properly for a change and left half the references to account updates in place:
> ![account_updates](https://user-images.githubusercontent.com/6610056/102157196-e9b2de80-3e33-11eb-9c19-11549fb229cb.png)
> 
> Un-be-lie-va-ble.

Hello,

The section was removed as there are not any actual messages to be streamed from that subscription channel
